dotnet-audit-logs-example/Views/Home/SendEvents.cshtml

```diff
@@ -3,105 +3,209 @@
+                                <div>
+                                    <a href="#send-events" class="remove-style"><button class="remove-style">Send Events</button></a>
+                                </div>
                             </div>
-                        </div>
-                        <div>
-                            <a href="/export_events">
-                                <div class="button button-outline width-15vw">
-                                    <p>Export Events</p>
+                            <div data-tab-target="#export-events" name='#export-events' class="flex space-evenly width-11vw content-button  tab">
+                                <div><i icon-name="download" class="stroke_width=1"></i></div>
+                                <div>
+                                    <a href="#export-events" class="remove-style"><button class="remove-style">Export Events</button></a>
+                                </div>
+                            </div>
+                            <div class="flex space-evenly width-11vw content-button tab">
+                                <div><i icon-name="eye" class="stroke_width=1"></i></div>
+                                <div>
+                                    <a href='@string.Format("/events/{0}", "AuditLogs")' class="remove-style"><button class="remove-style">View Events</button></a>
                                 </div>
-                            </a>
+                            </div>
+                            <div class="flex space-evenly width-11vw content-button tab">
+                                <div><i icon-name="share-2" class="stroke_width=1"></i></div>
+                                <div>
+                                    <a href='@string.Format("/events/{0}", "LogStreams")' class="remove-style"><button class="remove-style">Configure Log Streams</button></a>
+                                </div>
+                            </div>
                         </div>
-                    </div>
-                    <hr style="height:0.5px;width:100%;">
-                    <form action="/send_event" method="POST">
-                        <div class="flex margin-y-15">
-                            @using (Html.BeginForm("SendEvents", "Home", FormMethod.Post))
-                            {
+                        <div class="tab-content">
+                            <div id="send-events" name='#send-events' data-tab-content class="flex_column">
+                            <div>
+                                <h3>Send Events</h3>
+                            </div>
+                            <hr style="height:0.5px;width:100%;margin-bottom:25px;">
+                            <form id="send-event-form" method="post" action="/send_event">
+                                <div class="event-form-input-group flex-column flex-start">
+                                    <div><label for="event-action">Event Action</label></div>
```

Comment:
Could we leave off the event action and hard code it like we did with the other languages? It makes it easier for folks to get up and running if there's just one event to configure in their dashboard. This is already an error prone section, so I wanted it to be as foolproof as possible. 
![CleanShot 2023-03-30 at 16 02 54](https://user-images.githubusercontent.com/47123183/228974253-35f4e02a-5aa7-4b99-9ee7-79bfc1f5bdf2.png)